project-resources/MEMBER_TRAVEL_FUND.md

```diff
@@ -160,13 +163,13 @@ So that we can reimburse you more quickly, we strongly encourage you to use
 
 * A free [Expensify](https://www.expensify.com/) account is sufficient.
 * You can use the Android and iOS apps to capture receipts while you are
-  traveling.  If you do not use the app, you will need to upload photos of your
+  traveling. If you do not use the app, you will need to upload photos of your
   receipts after you return.
 * When you add an expense, please use local currency so that it matches the
   amount on the receipt.
 * Once you have uploaded your receipts, please add them to a report and submit
   it with [operations@openjsf.org](mailto:operations@openjsf.org) on cc, and the
-  Memo field set to "YOURNAME - TRIPNAME".
+  Memo field set to "YOURNAME-TRIPNAME".
```

Comment:
Is this change intentional?

Reply:
Thank you I have already fixed those in the last commit

Reply:
these are still not fixed.

Reply:
thats weird on my end it shows as fixed 
<img width="719" alt="Screenshot 2020-03-24 at 1 55 52 PM" src="https://user-images.githubusercontent.com/5986856/77460409-65b55400-6dd7-11ea-9f83-ab1b80cef8e9.png">

Reply:
it's still not fixed - there should be spaces around the hyphen; in your screenshot, there is not.
```suggestion
  Memo field set to "YOURNAME - TRIPNAME".
```

Reply:
Added the fix i see thanks